src/test/lib/logical_query_plan/sort_node_test.cpp

```diff
@@ -62,9 +69,11 @@ TEST_F(SortNodeTest, HashingAndEqualityCheck) {
 TEST_F(SortNodeTest, Copy) {
   EXPECT_EQ(*_sort_node->deep_copy(), *_sort_node);
 
-  const auto sort_b = SortNode::make(
-      expression_vector(_a_d, _a_f, _a_i),
-      std::vector<SortMode>{SortMode::Descending, SortMode::Ascending, SortMode::Descending}, _table_node);
+  const auto sort_b =
```

Comment:
For the LQP tests, we should easily be able to test NULLS LAST as well, right?

Reply:
Added occurrences of NULLs last for description and equality